### DISCUSSION TITLE
要开学力

### DISCUSSION THREAD

**Discussion by jostar9527** [share121/inter-knot#2564] [其他]
![cg(25)](https://github.com/user-attachments/assets/7c3522a1-b544-4c5b-999e-7555dfaedf21)
作业欠了一堆。。。
哦，未曾谋面的陌生人，到时候还请来参加我的葬礼<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by 00zzxx** [comment]
相信自己！会有奇迹发生的！<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Reply by jostar9527** [reply]
哦，谢谢你！善良的陌生人！但我仿佛已经窥见了通往天国的道路！

**Comment by BHGD-031212** [comment]
一个晚上，一个奇迹，相信自己的潜能。<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/a785e557e39e100ce93d80508f112615_3814701686582051674.png" alt="素裳 冲鸭" width="75" >

**Reply by jostar9527** [reply]
谢谢！我突然发现我睡懒觉和熬夜的潜能非常大！
